src/app/content/components/Attribution.spec.tsx

```diff
@@ -165,13 +165,9 @@ describe('Attribution', () => {
       }) as any) as AppState;
 
       store = createTestStore(newState);
-      const { node } = renderToDom(render());
-      const details = node;
+      const { node: details } = await renderToDom(render());
```

Comment:
Actually `yarn test:unit` works for me even without this `await` statement so you are probably okay with removing this